docs/README.md

```diff
@@ -21,68 +25,60 @@ The Customers project focuses on solving authentic identification of humans who
-[./migrations](./migrations).
-- `title` should describe action of the migration, e.g.,
-  `create_accounts_table`, `add_name_to_accounts`.
+Migrations are stored as files in the [/migrations](./migrations) directory. Contents of each file are executed by the database driver. The migration files should consist of valid SQL queries. The file names must adhere to the format: `{version}_{title}.up.sql`
 
+- `verision` of the migration should be represented as an integer with 3 digits (with leading zeros: e.g., 007). The migrations are applied in ascending order based on the version numbers. You can find examples of different migrations in [./migrations](./migrations).
+- `title` should describe what the migration is doing, e.g., `create_accounts_table`, `add_name_to_accounts`.
 
 ### Embedding Migrations
 
-We use [pkger](https://github.com/markbates/pkger) to embed migration files
-into our application. Please, [install
-it](https://github.com/markbates/pkger#installation) before you proceed.
+We use [pkger](https://github.com/markbates/pkger) to embed migration files into our application. Please [install it](https://github.com/markbates/pkger#installation) before you proceed.
 
-Running `make embed-migrations` will generate `cmd/server/pkged.go` file with
-encoded content of `/migrations` directory that will be included into
-application build. Please, commit generated file to the git repository.
+Running `make embed-migrations` will generate a `cmd/server/pkged.go` file with the encoded contents from the `/migrations` directory which will be included into application build. Make sure to commit the generated file to the git repository.
 
 ## Getting Help
-
  channel | info
  ------- | -------
-[Project Documentation](https://docs.moov.io/) | Our project documentation available online.
-Twitter [@moov_io](https://twitter.com/moov_io)	| You can follow Moov.IO's Twitter feed to get updates on our project(s). You can also tweet us questions or just share blogs or stories.
-[GitHub Issue](https://github.com/moov-io/customers/issues) | If you are able to reproduce a problem please open a GitHub Issue under the specific project that caused the error.
-[moov-io slack](https://slack.moov.io/) | Join our slack channel to have an interactive discussion about the development of the project.
+[Project Documentation](https://docs.moov.io/) | Documentation of Moov's various open source projects.
+Issues [GitHub Issue](https://github.com/moov-io/customers/issues) | If you are able to reproduce a problem, please open a GitHub Issue under the project that caused the error.
+Twitter [@moov_io](https://twitter.com/moov_io)	| You can follow Moov's Twitter feed to get updates on our projects. You can also tweet us to ask questions or share comments.
+Slack [#moov-io](https://slack.moov.io/) | Join the slack channel to discuss with other contributors about the development of Moov's open source projects.
```

Comment:
This footer is used in every project. If we're going to change it can we do so in the template first?